recipe/meta.yaml

```diff
@@ -19,12 +20,14 @@ build:
 
 requirements:
   host:
-    - pip >=18.0
-    - python >=3.6
+    - pip >=22.0.4
```

Comment:
`pip` should be without pinning in `host`, see https://github.com/pypa/pipenv/blob/14bc6de4c0fde7fcfc4b3945bdb1d9b61e500944/setup.py#L74